### PR TITLE
Node duplication bug reproduction

### PR DESCRIPTION
**Pull request by Fabs**
Most of the discussions on how to reproduce this one is here:
https://jira.mesosphere.com/browse/DCOS-20507

image

### CODE REVIEW

**Comment by Fabs** [comment]
image

**Comment by Fabs** [comment]
https://cl.ly/0u1n0Z0D3P1j

**Comment by juliangieseke** [comment]
some context? :) 
what exactly is the issue here? 
by pushing the first item, you basicly duplicate it. 
so the issue is that is is displayed twice?

File: plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js [hunk 1]
```diff
@@ -27,6 +27,15 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
       },
       { name: "state", events: ["success"], suppressUpdate: false }
     ];
+
+    setInterval(this.messIt.bind(this), 1000);
+  }
+
+  messIt() {
+    const items = this.state.filteredNodes.list;
```
Comment (weblancaster) [inline plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js]: Instead `this.state.filteredNodes.list` I would suggest `this.state.filteredNodes.list.slice(0)` that way avoiding shallow copy.

Comment (Fabs) [reply plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js]: I think this is intended, or at least it helps me to cause the bug I want to make happen. I will try to remove and see if it causes the bug.

**Comment by Fabs** [comment]
On the second commit, this is a weird behavior, where it shows the Composite Store seeing two different truths at the same time.
image

**Pull request closed without merging**